spec/models/course_spec.rb

```diff
@@ -94,6 +94,66 @@
         end
       end
 
+      describe '#current_level_threshold' do
+        subject do
+          -> (course, experience_points) { course.current_level_threshold experience_points }
```

Comment:
Quick qn: 

There are two possible types of course, one with and one without levels, so I use context to split them up. In both cases, the subject is the same which was why I used a lambda. 

For the one with levels, I'm using the `course` variable defined in the parent block, while I had to create a separate course without levels. 

So to check, when you say to make course a let variable, does it mean for me to rebuild another course, or explicitly restate the course (eg. `let(:course_with_levels) { course }`)? Or am I missing something here?